app/controllers/users_controller.rb

```diff
@@ -1,4 +1,25 @@
 class UsersController < ApplicationController
+  def show
+    @user = User.find(params[:id])
+  end
+
   def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      flash[:success] = "Welcome to the Sample App!"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.